packages/forklift-console-plugin/src/modules/Providers/views/details/tabs/VirtualMachines/OpenShiftVirtualMachinesRow.tsx

```diff
@@ -13,21 +12,15 @@ import { getVmTemplate } from './utils';
 const toNamespace = ({ data }: VMCellProps) =>
   (data.vm.providerType === 'openshift' && data.vm.object?.metadata?.namespace) || '';
 
-const toGVK = ({ data }) =>
-  (data.vm.providerType === 'openshift' && groupVersionKindForObj(data.vm.object)) || {
```

Comment:
did it fail for some data?

Reply:
yes, on my cluster :-)

Reply:
more information on the partial info:
vm object contain:
```
apiVersion: kubevirt.io/v1
kind: VirtualMachine
metadata: {..}
spec: {...}
status: {...}
```

but Inventory server may return only partial data:
```
metadata: {..}
spec: {...}
status: {...}
```

without the `apiVersion` field, it's ok, because we know it's a vm, but the code need to handle this eventuality
about  `apiVersion`  changes, and being future prof, we know what versions of kubevirt we support, so we are safe hardcoding the `apiVersion`